c10/core/CPUAllocator.cpp

```diff
@@ -75,9 +75,6 @@ ProfiledCPUMemoryReporter& profiledCPUMemoryReporter() {
 template <uint32_t PreGuardBytes, uint32_t PostGuardBytes>
 class DefaultMobileCPUAllocator final : public at::Allocator {
  public:
-  DefaultMobileCPUAllocator() = default;
```

Comment:
Why delete these?

Reply:
Because they repeat the parent class methods without new code.